src/Foundation/NSXpcConnection.cs

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright 2009-2010, Novell, Inc.
+// Copyright 2011, 2012 Xamarin Inc
+//
```

Comment:
Xamarin is no more.. and I wonder, is this a new file? Then Novell and Xamarin do not make much sense, right?